blocks/melody/ff_container/ff_container-expandable/ff_container-expandable.less

```diff
@@ -1,29 +1,36 @@
-.ff_container-expandable__dropdown {
-	padding: @ff_size_spacing_med;
-	border: 1px solid @ff_colour_melody_int_blue;
-	color: @ff_colour_melody_int_blue;
-	cursor: pointer;
-	background: white;
-}
+.ff_container-expandable {
+	&__header {
+		padding: @ff_size_spacing_med;
+		border: 1px solid @ff_colour_melody_int_blue;
+		color: @ff_colour_melody_int_blue;
+		cursor: pointer;
+		background: white;
+	}
 
-.ff_container-expandable__dropdown--small-screens {
-	@media all and (min-width: @ff_bp_medium) { 
-		display: none;
+	&__icon {
+		width: 1.5em;
+		height: 1.5em;
+		float: right;
 	}
-}
 
-.ff_container-expandable__icon  {
-	width: 1.5em;
-	height: 1.5em;
-	float: right;
-}
+	&__content {
+		max-height: 1000px;
```

Comment:
I am not sure we are talking about the same thing. This is not the filters, that was merged by Danielle and gives the option to have a max-height through a modifier (as we discussed) and then a scroll bar to scroll the filters. The expandable component needs a high max-height to do the sliding animation (so we set it to a high value in this case 1000px). Should we have a higher value for the max-height @dkhuntrods ?